src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudServiceActor.scala

```diff
@@ -0,0 +1,69 @@
+package org.broadinstitute.dsde.firecloud
+
+import akka.actor.ActorLogging
+import com.gettyimages.spray.swagger.SwaggerHttpService
+import com.wordnik.swagger.model.ApiInfo
+import org.broadinstitute.dsde.firecloud.service.MethodsService
+import spray.http.StatusCodes._
+import spray.routing.HttpServiceActor
+
+import scala.reflect.runtime.universe._
+
+class FireCloudServiceActor extends HttpServiceActor with ActorLogging {
+
+  // the HttpService trait defines only one abstract member, which
+  // connects the services environment to the enclosing actor or test
+  override def actorRefFactory = context
+
+  trait ActorRefFactoryContext {
+    def actorRefFactory = context
+  }
+
+  val methodsService = new MethodsService with ActorRefFactoryContext
+
+  private implicit val ec = context.dispatcher
```

Comment:
as an implicit, you might not be able to rename ... but I like "executionContext" better than "ec" for clarity.